lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -401,9 +401,9 @@ template <> struct less<swift::ClusteredBitVector> {
     for (; iL >= 0 && iR >= 0; --iL, --iR) {
```

Comment:
why does this even exist? Are we putting them into a std::map?

Reply:
I couldn't answer that - I don't know why it exists, Im just trying to avoid extension usage.  We could try removing it separately I suppose.

Reply:
That was more a question for the room — I know you're just the messenger :-)